reggie_config/super_2020_prod/init.yaml

```diff
@@ -0,0 +1,13 @@
+__: merge-first
+
+reggie:
+  maintenance: True
+  onsite_redirect_host: onsite.reggie.magfest.org
+  plugins:
+    ubersystem:
+      config:
+        dev_box: True
+        send_emails: False
+        send_sms: False
+        at_the_con: False
+        post_con: False
```

Comment:
I switched our prod server into a dev_box and turned off emails/SMS messages because technically the background tasks are still running, so we don't want them doing dumb stuff like sending emails.